JuliaDatabases/ODBC.jl
# Big rewrite of package

Some of the big changes include:
  * This will be a breaking, 1.0 release
  * Use of BinaryBuilder for source driver managers
  * Streamlines resultset fetching process supporting super-efficient
  direct column fetching for native types
  * More robust/useful statement preparation and parameter binding by
  conforming to the `DBInterface` interface
  * Moving the package to own it's own config; the aim here is to cut
  down on all the crazy config stuff available across the various driver
  managers and attempt to keep it close and sane; additional methods for
  adding/getting/deleting drivers and datasources have been added
  * Enhance debugging capabilities via `ODBC.setdebug` and passing
  `debug=true` when executing statements.

Things still to do:
  * test new driver/dsn management methods
  * ensure tests pass across all platforms
  * little more parameter binding testing
  * Update index.md docs (internal docs have been updated)
  * Update tests for various platforms
